public/templates/book.php

```diff
@@ -1,31 +1,34 @@
 <?php
 
-use App\CreationLog;
 use App\FontProvider;
 use App\GeneratorSelector;
 use App\Util\Api;
 
 include 'header.php';
 ?>
+<ol class="breadcrumb">
+	<li class="active">Home</li>
+	<li><a href="stat.php">Statistics</a></li>
+</ol>
 <div class="row">
```

Comment:
Now that there's only one column, this whole row element could be removed (and the col specification below). Not that it really matters.

Reply:
Done.